e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -574,6 +574,24 @@ public async Task RegistryManager_DeviceTwinLifecycle()
             }
         }
 
+        [LoggedTestMethod]
+        public async Task DevicesClient_GetStatistics()
+        {
+            using var serviceClient = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionString);
+
+            // No great way to test the accuracy of these statistics, but making the request successfully should
+            // be enough to indicate that this API works as intended
+            ServiceStatistics serviceStatistics = await serviceClient.Devices.GetServiceStatisticsAsync().ConfigureAwait(false);
+            serviceStatistics.ConnectedDeviceCount.Should().BeGreaterOrEqualTo(0);
+
+            // No great way to test the accuracy of these statistics, but making the request successfully should
+            // be enough to indicate that this API works as intended
+            RegistryStatistics registryStatistics = await serviceClient.Devices.GetRegistryStatisticsAsync().ConfigureAwait(false);
+            registryStatistics.DisabledDeviceCount.Should().BeGreaterOrEqualTo(0);
```

Comment:
I worry that things like this will cause the test to fail, depending on the condition of the hub.

If the test doesn't guarantee devices exist in this state, I think we should be wary of asserting on them. I'd say, consider adding these devices then cleaning up afterwards, or simply assert the requests returns without incident (exception) and that the properties exist.

Reply:
> or simply assert the requests returns without incident (exception) and that the properties exist.

That's what I'm doing here. The only real assertion in this test is that the values are greater than or equal to 0. There is no condition where a hub has negative disabled devices

Reply:
Okay, fair enough.